src/Unit/AbstractUnit.php

```diff
@@ -158,6 +157,19 @@ public function isEnabled(): bool
 
         return trim($output) === 'enabled';
     }
+    
+    /**
+     * Get the raw (text) output of the `is-enabled` command.
+     *
+     * @return string
+     */
+    public function isEnabledRaw(): string
+    {
+        // We have to trim() the output, as it may end in a newline character that we don't want.
+        $output = \trim($this->execute('is-enabled')->getOutput());
```

Comment:
Great thanks. One thing tho, can you remap the `isActive` and `isEnabled` method to use the output of `isActiveRaw` and `isEnabledRaw` so we can reduce code duplication.

Reply:
Good point! Done.